faqs/galaxy/datasets_create_new_file.md

```diff
@@ -1,3 +1,11 @@
+---
+title: Creating a new file
+description: Galaxy allows you to create new files from the upload menu. You can supply the contents of the file.
+area: datasets
+
```

Comment:
Some metadata missing and also need to remove the tip box below

Reply:
oops, missed that one, thanks for spotting!